contracts/StakingRewards.sol

```diff
@@ -357,4 +360,23 @@ contract StakingRewards is
             }
         }
     }
+
+    /** @dev Withdrawing rewards acumulated from different pools for providing liquidity
+    * @param recipient The address to whom the rewards will be trasferred
+    * @param lpTokenContract The address of the rewards contract
+    */
+    function withdrawLPRewards(address recipient, address lpTokenContract)
+        public
+        nonReentrant
+        onlyOwner {
```

Comment:
Instead of adding onlyOwner and bloat the code with `Ownable` in general, I would use `onlyRewardsDistributor` and add code in the factory to trigger it.